From: Luis R. Rodriguez <mcgrof@do-not-panic.com>
Date: Wed, 23 Oct 2013 14:55:36 -0400
Subject: [RFC] cfg80211: make genregdb.awk skip antenna gain

Now that wireless-regdb doesn't include
antenna gain lets skip parsing it completely
for when CONFIG_CFG80211_INTERNAL_REGDB is
enabled.

Signed-off-by: Luis R. Rodriguez <mcgrof@do-not-panic.com>
---
 net/wireless/genregdb.awk | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)
 mode change 100644 => 100755 net/wireless/genregdb.awk

--- a/net/wireless/genregdb.awk
+++ b/net/wireless/genregdb.awk
@@ -56,14 +56,11 @@ function parse_reg_rule()
 	end = $3
 	bw = $5
 	sub(/\),/, "", bw)
-	gain = $6
-	sub(/\(/, "", gain)
-	sub(/,/, "", gain)
-	power = $7
-	sub(/\)/, "", power)
+	power = $6
+	sub(/\(/, "", power)
 	sub(/,/, "", power)
 	# power might be in mW...
-	units = $8
+	units = $7
 	sub(/\)/, "", units)
 	sub(/,/, "", units)
 	dfs_cac = $9
@@ -86,7 +83,7 @@ function parse_reg_rule()
 	sub(/\(/, "", dfs_cac)
 	sub(/\)/, "", dfs_cac)
 	flagstr = ""
-	for (i=8; i<=NF; i++)
+	for (i=7; i<=NF; i++)
 		flagstr = flagstr $i
 	split(flagstr, flagarray, ",")
 	flags = ""
